src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +56,10 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+    @Override
+    public String getPrintableString() {
+        String header =  !isPrivate ? " Address: " : " Address: (private) ";
```

Comment:
interestingly you indicate if detail is private. You could have removed the `!` and change the order in the ternary operands. Also, you could put the operands as constants instead